toolkit/motors/rev_motors.py

```
@@ -66,6 +66,11 @@ def __init__(self, can_id: int, inverted: bool = False, brushless: bool = True,
         self._configs = []
 
         self._configs.append(config)
+        
+        if config_others is not None:
+            for config in config_others:
+                if isinstance(config, SparkMaxConfig):
+                    self._configs.append(config)
```

If it isn't SparkMaxConfig we should raise an error.